### PR TITLE
stops paper planes from spinning

### PR DESCRIPTION
**Pull request by PKPenguin321**
the comment on it says it's specifically to stop spinning but it fails to do that

### CODE REVIEW

**Comment by ChangelingRain** [comment]
33% chance oranges will murder you

**Comment by PKPenguin321** [comment]
why

File: code/modules/paperwork/paperplane.dm [hunk 1]
```diff
@@ -87,7 +87,7 @@
 	add_fingerprint(user)
 
 
-/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0) //prevent the paper plane from spinning
+/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=0, diagonals_first = 0) //prevent the paper plane from spinning
```
Comment (optimumtact) [inline code/modules/paperwork/paperplane.dm]: This won't do anything, the spin value is being passed in from further up the call chain, most likely wherever the item is actually being thrown from (mob throw code).

Setting it here only sets the *default* value when the spin argument is not specified in higher calls.

The right way to force a specific value to always be one thing, no matter the original calling value

Don't include my comment without shortening it by the way it's just to indicate the code
```

/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=FALSE, diagonals_first = FALSE)
    //call the parent and explicitly set spin to be false, passing all other values in untouched
    . = ..(target, range, speed, thrower, spin=FALSE, diagonals_first)
```

**Comment by PKPenguin321** [comment]
Woops I forgot I had this PR

I'll look at it tomorrow

**Comment by phil235** [comment]
@PKPenguin321

**Comment by PKPenguin321** [comment]
@optimumtact how's that

**Comment by PKPenguin321** [comment]
?????

**Comment by Jordie0608** [comment]
This runtimes

> runtime error: 
[15:48:30]positional parameters must precede all named args
[15:48:30]proc name: throw at (/obj/item/throw_at)
[15:48:30]  usr: Oui Oui (/mob/living/carbon/human)
[15:48:30]  src: the paper plane (/obj/item/weapon/paperplane)
[15:48:30]  usr.loc: the floor (133,142,1) (/turf/open/floor/wood)
[15:48:30]  src.loc: the floor (133,142,1) (/turf/open/floor/wood)
[15:48:30]  call stack:
[15:48:30]the paper plane (/obj/item/weapon/paperplane): throw at(the floor (138,142,1) (/turf/open/floor/wood), 7, 1, Oui Oui (/mob/living/carbon/human), 0, null)
[15:48:30]the paper plane (/obj/item/weapon/paperplane): throw at(the floor (138,142,1) (/turf/open/floor/wood), 7, 1, Oui Oui (/mob/living/carbon/human), 0, 0)
[15:48:30]Oui Oui (/mob/living/carbon/human): throw item(the floor (138,142,1) (/turf/open/floor/wood))
[15:48:30]Oui Oui (/mob/living/carbon/human): ClickOn(the floor (138,142,1) (/turf/open/floor/wood), "icon-x=8;icon-y=13;left=1;scre...")
[15:48:30]the floor (138,142,1) (/turf/open/floor/wood): Click(the floor (138,142,1) (/turf/open/floor/wood), "mapwindow.map", "icon-x=8;icon-y=13;left=1;scre...")

**Comment by PKPenguin321** [comment]
Ok so I did it wrong I guess send help

File: code/modules/paperwork/paperplane.dm [hunk 2]
```diff
@@ -87,9 +87,8 @@
 	add_fingerprint(user)
 
 
-/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=1, diagonals_first = 0) //prevent the paper plane from spinning
-	if(!..())
-		return
+/obj/item/weapon/paperplane/throw_at(atom/target, range, speed, mob/thrower, spin=FALSE, diagonals_first = FALSE)
+	. = ..(target, range, speed, thrower, spin=FALSE, diagonals_first)
```
Comment (optimumtact) [inline code/modules/paperwork/paperplane.dm]: . = ..(target, range, speed, thrower, FALSE, diagonals_first)

you can't specify a named arg before a positional one (i.e after one argument of the form x=y, all future arguments must also be x=y

the other option is

    . = .. (target, range, speed, thrower, spin=FALSE, diagonals_first=diagonals_first)

but this is more confusing

**Comment by PKPenguin321** [comment]
okay how about now

**Pull request merged**